packages/ai-openai/src/browser/openai-preferences.ts

```diff
@@ -36,7 +36,19 @@ on the machine running Theia. Use the environment variable `OPENAI_API_KEY` to s
             type: 'array',
             description: nls.localize('theia/ai/openai/models/description', 'Official OpenAI models to use'),
             title: AI_CORE_PREFERENCES_TITLE,
-            default: ['gpt-4o', 'gpt-4o-2024-11-20', 'gpt-4o-2024-08-06', 'gpt-4o-mini', 'o1', 'o1-mini', 'o3-mini', 'gpt-4.5-preview'],
+            default: [
+                'gpt-4.1',
```

Comment:
I believe, if we put this on place one, it will be the global default (instead of gpt-4o). I think we should maybe not do this before having tested it a bit in practice. And we should introduce: https://github.com/eclipse-theia/theia/issues/15395
soon to make the default management easier.

Reply:
I'll move it to second place and yes the management ticket is important